beacon-chain/db/kv/state_summary.go

```diff
@@ -37,6 +49,25 @@ func (s *Store) SaveStateSummaries(ctx context.Context, summaries []*ethpb.State
 	return nil
 }
 
+func (s *Store) ensureBlocksSaved(ctx context.Context, blockCache func([32]byte) interfaces.SignedBeaconBlock) error {
+	if blockCache == nil {
+		return nil
+	}
+	blocks := make([]interfaces.SignedBeaconBlock, 0)
+	summaries := s.stateSummaryCache.getAll()
+	for _, ss := range summaries {
+		if s.HasBlock(ctx, bytesutil.ToBytes32(ss.Root)) {
+			log.WithField("blockRoot", fmt.Sprintf("%#x", ss.Root)).Debug("Block already saved")
```

Comment:
This log may be noisy even under debug. I'm not sure though...

Reply:
Yeah, i'll delete it